src/test/java/com/stripe/net/StripeHeadersTest.java

```diff
@@ -0,0 +1,55 @@
+package com.stripe.net;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.Stripe;
+import com.stripe.net.StripeHeaders;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StripeHeadersTest extends BaseStripeTest {
+
+	private Map<String, List<String>> generateHeaderMap () {
+		Map<String, List<String>> headerMap = new HashMap<String, List<String>>();
+		List<String> bearerHeader = new ArrayList<String>();
+		bearerHeader.add("Bearer");
+		bearerHeader.add("sk_live_12345");
+		List<String> requestIdHeader = new ArrayList<String>();
+		requestIdHeader.add("req_12345");
+		headerMap.put("Request-Id", requestIdHeader);
+		headerMap.put("Authorization", bearerHeader);
```

Comment:
I know that you're just using this header to test the `get()` method, but I find it a bit confusing to have an `Authorization` header in an object that represents the _response_'s header. Would you mind changing it to something else? Either a generic name e.g. `"Some-Header"` or a header that can actually be found in our responses e.g. `"Stripe-Version"`.